src/utils.js

```diff
@@ -41,3 +43,26 @@ export const removeEmptyKeys = (obj) => {
 export function sleep(ms) {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
+
+export const updateFlagImage = (barcode, src) => {
+  return axios.put(
+    `https://amathjourney.com/api/off-annotation/flag-image/${barcode}`,
+    {
+      mode: "no-cors",
+      imgid: Number(src[src.length - 5]),
+      url: src,
+    }
+  );
+};
+
+export const removeFlagImage = (barcode, src) => {
```

Comment:
maybe would be useful to have a comment of what means removing or updating a flag, just to give some context of the _why_ of this operation